### PR TITLE
Public Cloud Tools CLI: Fix Azure test and scheduling

### PR DESCRIPTION
**Pull request by pdostal**
- Related merge request: [qac/qac-openqa-yaml!546](https://gitlab.suse.de/qac/qac-openqa-yaml/-/merge_requests/546)
- Verification run: TBD

### CODE REVIEW

File: tests/publiccloud/azure_cli.pm [hunk 1]
```diff
@@ -36,17 +36,21 @@ sub run {
...
+    my $openqa_ttl = get_var('MAX_JOB_TIME', 7200) + get_var('PUBLIC_CLOUD_TTL_OFFSET', 300);
+    my $created_by = get_var('PUBLIC_CLOUD_RESOURCE_NAME', 'openqa-vm');
+    my $tags = "openqa-cli-test-tag=$job_id openqa_created_by=$created_by openqa_ttl=$openqa_ttl";
+
+    # Configure default location and create Resource group
     assert_script_run("az configure --defaults location=southeastasia");
-    assert_script_run("az group create -n $resource_group");
+    assert_script_run("az group create -n $resource_group --tags '$tags'");
 
+    # Pint
```
Comment (asmorodskyi) [inline tests/publiccloud/azure_cli.pm:47]: well I would do one of two things : 
1. drop this comment 
2. add something more detailed like `Pint - command line tool to query pint.suse.com ( info about ALL images in ALL CSP's managed by SUSE) `
In current form the one who know what is pint don't need this line , the one who don't know what is pint can not get any useful info from this line

**Review by asmorodskyi — APPROVED** [review]
beside very minor thing with comment LGTM

**Comment by asmorodskyi** [comment]
https://gitlab.suse.de/qac/qac-openqa-yaml/-/merge_requests/546 - already merged

File: tests/publiccloud/azure_cli.pm (hunk 1 above)
Comment (pdostal) [reply tests/publiccloud/azure_cli.pm:47]: I changed it to:
```
# Pint - command line tool to query pint.suse.com to get the current image name
```

**Comment by okurz** [comment]
s/CLoud/Cloud/

**Review by jlausuch — APPROVED** [review]

**Comment by dzedro** [comment]
This VR is not real use case, the result is broken test in osd https://openqa.suse.de/tests/overview?version=15-SP3&groupid=370&flavor=PC-Tools-CLI&distri=sle&build=20211222-1

**Comment by jlausuch** [comment]
> This VR is not real use case, the result is broken test in osd https://openqa.suse.de/tests/7895112 due to bad HDD_1 https://openqa.suse.de/tests/overview?version=15-SP3&groupid=370&flavor=PC-Tools-CLI&distri=sle&build=20211222-1 https://suse.slack.com/archives/C02CGKBCGT1/p1640180264310300

The problem is different. See my reasoning here: https://gitlab.suse.de/qac/qac-openqa-yaml/-/merge_requests/562
Basically, wrong flavor vs dependency :) 
I really don't like that openQA can't schedule dependencies between different flavors... it would make our lives much easier... specially in public cloud...

**Comment by dzedro** [comment]
It can but you have to use same HDD_1 which was published by job from another flavor.

**Comment by asmorodskyi** [comment]
> It can but you have to use same HDD_1 which was published by job from another flavor.

hm can you point to some example in osd where job has `START_AFTER_TEST` which is job having different `FLAVOR` ? I would say it is just impossible because job dependencies are calculated during `isos post` and `isos post` can not handle two `FLAVOR` variables by design

**Comment by dzedro** [comment]
For example the test based on which I commented here :)
I didn't do it by myself, but Marcelo had such issue with windows and SLE
If I remember correctly it is tricky but possible, you have to use + on variable so it will overwrite the flavor despite you run `post` with other flavor.

https://openqa.suse.de/tests/7892942 qcow2 created
https://openqa.suse.de/tests/7895652 created qcow2 used with different flavor

**Comment by pdostal** [comment]
@asmorodskyi we use this in [container aggregates](https://openqa.suse.de/tests/7934768).

**Pull request merged**